specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/sqlServer.json

```diff
@@ -1538,7 +1538,11 @@
         },
         "isStorageSecondaryKeyInUse": {
           "description": "Specifies whether storageAccountAccessKey value is the storage's secondary key.",
-          "type": "boolean"
+          "type": "boolean",
+          "x-ms-mutability": [
```

Comment:
Can you please share why you dont have read on this?

Reply:
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=47D73218-13A2-4087-80F3-81C934A057F7

ROUNDTRIP_MISSING_PROPERTY
The property is present in the PUT request but is either never returned in the GET response or is returned with a null value. If this is a property that carries a secret such as a password, update the Swagger definition to mark it with the "x-ms-secret": true annotation. If it is write-only but does not carries a secret, update the swagger definition for this property to mark it with the "x-ms-mutability": ["create", "update"] annotation.